airbyte-scheduler/build.gradle

```diff
@@ -3,6 +3,8 @@ plugins {
 }
 
 dependencies {
+    implementation 'io.temporal:temporal-sdk:1.0.4'
```

Comment:
question for my own understanding: we place external dependencies above internal ones?

Reply:
i think we actually usually do internal and then external.

(within each group we alphabetize)
```
main internal

main external

test internal

test external

integrationTest internal

integrationTest external
```